pvlib/modelchain.py

```diff
@@ -330,10 +330,15 @@ class ModelChain:
         'interp' and 'no_loss'. The ModelChain instance will be passed as the
         first argument to a user-defined function.
 
-    spectral_model : str, or function, optional
+    spectral_model : str or function, optional
         If not specified, the model will be inferred from the parameters that
-        are common to all of system.arrays[i].module_parameters.
-        Valid strings are 'sapm', 'first_solar', 'no_loss'.
+        are common to all of system.arrays[i].module_parameters. Valid strings
```

Comment:
I think I would put the "Valid strings are..." before the "If not specified..." part.